sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -253,6 +256,15 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
           submittedDate: now,
           offering: { id: savedOffering.id } as EducationProgramOffering,
         } as StudentAssessment;
+        // Update the student appeal record for the student assessment if it exists.
+        if (application.currentAssessment?.studentAppeal) {
```

Comment:
Similar to https://github.com/bcgov/SIMS/pull/2961/files#r1518279300
Also keep in mind that `application.currentAssessment` would not need a null check at this stage.